examples/dt/uni04delta-ipv6/control-plane/service-values.yaml

```diff
@@ -8,9 +8,6 @@ metadata:
 data:
   preserveJobs: false
 
-  horizon:
-    enabled: false
```

Comment:
we just need to change this to true

Reply:
(see https://github.com/openstack-k8s-operators/architecture/pull/341/files#r1687774153)